auto_nag/scripts/b2g_query_creator.py

```diff
@@ -104,11 +105,15 @@ def createQueriesList(print_all, queries_dir):
 ]
 
 
-def cleanUp():
-    for file in os.listdir(queries_dir):
-        if file.startswith(str(datetime.date.today())):
-            os.remove(os.path.join(queries_dir, file))
-
+def cleanUp(queries_dir):
+    try:
+        for file in os.listdir(queries_dir):
+            if file.startswith(str(datetime.date.today())):
```

Comment:
What happens if the tests start at 23:59:58 and we change day during the test?
temporary files will remain on the system?